baseapp/abci.go

```diff
@@ -828,7 +828,31 @@ func handleQueryStore(app *BaseApp, path []string, req abci.RequestQuery) abci.R
 	resp := queryable.Query(req)
 	resp.Height = req.Height
 
+	if path[1] == "hashes" {
```

Comment:
nit: can we put this into a const?

Reply:
They should all be `const` IMO. Speaking of which, what does `hashes` mean? Should it be `commit_info` or something?

Reply:
You're right, store_infos would probably be a better name, changing!